### PR TITLE
WIP first test pass, create registration controller for username field d...

### PR DESCRIPTION
**Pull request by TeriPastorino**
...evise
@levicole 
went ahead and added information for a username field per email i sent you
first test works, took out should contain the end

still confused on continued working flow. can i just be in this branch and do checkout -b here ? will it copy this branch for me to continue to work on it under a new name?

i am off to research it as well

### CODE REVIEW

**Comment by levicole** [comment]
You can actually just stay in this branch for a while until we get all the tests written and passing for the registration/signin flow. So after each commit, you will have to:

```
git branch #make sure you are on the correct branch (rspec-capybara)
git push origin rspec-capybara
```

If you run into any problems with this, let me know. There is one case where git will reject the push, but that shouldn't happen because you are the only person working on this branch.

**Comment by TeriPastorino** [comment]
Ok great thanks

File: spec/features/user_registration_spec.rb [hunk 1]
```diff
@@ -1,54 +1,55 @@
...
+
+        expect(current_path).to eq root_path
+        expect(page).to have_content('activate')
+    end
   end
-end
 
-describe "signup flow" do
+  describe "authenticate" do
+  #  before do
```
Comment (levicole) [inline spec/features/user_registration_spec.rb]: You need to wrap 27 to 35 in an it block. That will fix the error you are receiving.

File: spec/features/user_registration_spec.rb [hunk 2]
```diff
@@ -1,54 +1,81 @@
...
-    visit '/new_user'
-    page.should have_content ('User Registration')
+  describe "signup page" do
+    it "should have the content 'Sign up'" do
+      visit new_user_registration_path
+      expect(page).to have_content('Sign up')
+    end
+  end
+  
+# THIS TEST IF FAILING - it says users already exists - set up user factory? and use database cleaner to test???
```
Comment (levicole) [inline spec/features/user_registration_spec.rb]: I think you are right. Let's set up database cleaner and factory girl today!

**Comment by levicole** [comment]
This is looking good so far! We can fill out the pending specs today in our meeting.

**Comment by TeriPastorino** [comment]
great talk to you in a little bit 

i installed factory girl, per the bloccit instructions so you should see that in the push 

if there is anything i am missing let me know 
----- Original Message -----

**Pull request merged**